nk/main.go

```diff
@@ -157,7 +157,7 @@ func sign(fname, keyFile string) {
 	if err != nil {
 		log.Fatal(err)
 	}
-	log.Printf("%s", base64.StdEncoding.EncodeToString(sigraw))
+	log.Printf("%s", base64.RawURLEncoding.EncodeToString(sigraw))
```

Comment:
Why the change here?

Reply:
Because in JWT three parts, header.payload.signature, the signature part is encoded in base64 url no padding format according JWT spec. If `nk` uses base64 url no padding format as signature content,  `nk` command can be used to verify jwt token like this: 
    nk -verify header.payload -pubin pubin -sigfile signature
Or nk would compain illegle charset error.